src/ClientCommand.cpp

```diff
@@ -635,20 +635,19 @@ void CClient::UserCommand(CString& sLine) {
 			return;
 		}
 
-		const CModules& vMods = pOldNetwork->GetModules();
-		for (CModules::const_iterator i = vMods.begin(); i != vMods.end(); ++i) {
-			CString sOldModPath = pOldNetwork->GetNetworkPath() + "/moddata/" + (*i)->GetModName();
-			CString sNewModPath = pNewUser->GetUserPath() + "/networks/" + sNewNetwork + "/moddata/" + (*i)->GetModName();
+		for (auto pMod : pOldNetwork->GetModules()) {
```

Comment:
@md5 Things might seem clear to you at the time you refactor some piece of code, but think about people who have to read, maintain and browse the code later. The auto keyword is really neat when used as a replacement for clumsy long template/iterator monster types, but IMHO it shouldn't be used as the default type for every single for-loop, because it makes things ambiguous and less readable.